integration/query_array_compat_test.go

```diff
@@ -0,0 +1,260 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"math"
+	"testing"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+func TestQueryArrayCompatSize(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]queryCompatTestCase{
+		"float64": {
+			filter:        bson.D{{"v", bson.D{{"$size", float64(2)}}}},
+			skipForTigris: "https://github.com/FerretDB/FerretDB/issues/908",
```

Comment:
I think it might be easier to skip the whole test for Tigris, not all individual test cases.

Reply:
Let's do it that way :+1: